glide.yaml

```diff
@@ -20,7 +20,7 @@ import:
 - package: github.com/FactomProject/ed25519
   version: master
 - package: github.com/FactomProject/factom
-  version: develop
+  version: db9756aef1c3d6847a47b78ac0022bfcfac125b7
```

Comment:
how do we usually cope w/ this - will we merge the factom repo - then revert this ?

Reply:
once we have the factom lib changes in develop we can revert this change in walletd

Reply:
reverted and fixed

```
$ glide cc
$ glide update
$ glide install
```